dabuild.in

```diff
@@ -51,22 +58,21 @@ if [ ! \( -d "$ABUILD_PACKAGES" -a -w "$ABUILD_PACKAGES" \) ]; then
 fi
 
 ## setup volumes; use named volumes as cache if desired
-printf -v ABUILD_VOLUMES "%s "\
-       "-v ${HOME}/.abuild:/home/builder/.abuild" \
-       "-v ${PWD%/aports*}/aports:/home/builder/aports" \
-       "-v ${ABUILD_PACKAGES}:/home/builder/packages"
-
-if [ -s "/etc/alpine-release" ]; then
-  printf -v ABUILD_VOLUMES "%s "\
-         $ABUILD_VOLUMES \
-         "-v /etc/abuild.conf:/etc/abuild.conf"\
-         "-v /var/cache/distfiles:/var/cache/distfiles"\
-         "-v /var/cache/apk:/var/cache/apk"
```

Comment:
Is the idea here to save `/etc/apk/cache` in a volume so that downloads do not happen repeatedly?

Reply:
Yes thats the idea.
We can use a volume or mount it via bind mount.
Currently its broken as apk doesn't like the hosts apkindex cache.